.gitpod/composer.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "joomla/joomla-cms",
+  "type": "project",
+  "description": "Joomla CMS",
+  "keywords": ["joomla", "cms"],
+  "homepage": "https://github.com/joomla/joomla-cms",
+  "license": "GPL-2.0-or-later",
+  "config": {
+    "optimize-autoloader": true,
+    "platform": {
+      "php": "5.3.10"
+    },
+    "vendor-dir": "libraries/vendor",
+    "github-protocols": ["https"]
+  },
+  "support": {
+    "issues": "https://issues.joomla.org",
+    "irc": "https://irc.lc/freenode/joomla/",
+    "forum": "https://forum.joomla.org/",
+    "docs": "https://docs.joomla.org"
+  },
```

Comment:
## Note to self 

We were hitting rate limits for these backports:
https://github.com/joomla/joomla-cms/blob/staging/composer.json#L22-L33

By providing a modified composer.json file, using the bootstrap process:
https://github.com/joomlatools/joomlatools-pages-playground/pull/10/files#diff-6ed1d7f772b2017ba9cc5541f33c6ec1c4026442c0a15c85584f310c3462c119R43

We are able to effectively sidestep this problem